<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PencilProducer</class>
 <widget class="QMainWindow" name="PencilProducer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PencilProducer</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="makePencil">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #03A3FE;	
	color: #FFFFFF;
	border: 0px;
	font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string>Make Pencil</string>
    </property>
   </widget>
   <widget class="QLabel" name="inventory">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: #929292;
}</string>
    </property>
    <property name="text">
     <string>Pencils Produced (to date):</string>
    </property>
   </widget>
   <widget class="QLabel" name="wood_left">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>216</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wood Left:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buyWood">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #945213;	
	border: 0px;
	color: #FFFFFF;
	font-size: 14px;
}</string>
    </property>
    <property name="text">
     <string>Buy Wood</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buyGraphite">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #929292;	
	border: 0px;
	color: #FFFFFF;
	font-size: 14px;
}</string>
    </property>
    <property name="text">
     <string>Buy Graphite</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pencil Producer</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayWood">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1000.00m</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="graph_left">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Graphite Left:</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayGraph">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1000.00m</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayInvDate">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: #929292;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="inventoryLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>99</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pencils in Inventory:</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayInv">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minusButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #1FB200;	
	border: 0px;
	color: #FFFFFF;
	font-size: 15px;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plusButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #1FB200;	
	border: 0px;
	color: #FFFFFF;
	font-size: 15px;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="bankLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bank Balance</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayBalance">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: #1FB200;
}</string>
    </property>
    <property name="text">
     <string>$145.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="displayWoodPrice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>$1000 / 100m</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="displayGraphPrice">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>$1000 / 100m</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="autoPencil2000Label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AutoPencil2000:</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayAutoPencil2000">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="inventory_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: #929292;
}</string>
    </property>
    <property name="text">
     <string>Rate: 120 pencils/minute </string>
    </property>
   </widget>
   <widget class="QPushButton" name="buyMoreButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #03A3FE;	
	color: #FFFFFF;
	border: 0px;
	font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string>Buy More</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayAutoPencil200Price">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>$150.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="pencilPriceLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Price of Pencil:</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayPencilPrice">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>$1.00</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="publicDemandLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Public Demand:</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayPublicDemand">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>568</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
